pytest-wdl-tests/tests/expr/types/coercion/string_and_file/test_file2string_and_string2file_coercions.wdl

```diff
@@ -0,0 +1,37 @@
+version 1.0
+
+workflow test_file2string_and_string2file_coercions {
+    input {
+        String wf_in_file_name
+        String wf_in_file_content
+    }
+
+    call create_file { input: in_file_name = wf_in_file_name, in_file_content = wf_in_file_content }
+
+    String str_coerced_from_file = create_file.out_file     # File to String
+    File file_coerced_from_str = str_coerced_from_file              # String to File
+
+    output {
+        String out_str_coerced_from_file = sub(str_coerced_from_file, "^/(.+/)", "")
```

Comment:
Not 100% sure if this was in WDL 1.0 or 2.0, but there might be a `basename()` function that does the same thing as this `sub`